tests/TestCase/PaginationResultTest.php

```diff
@@ -15,6 +16,25 @@
 
 class PaginationResultTest extends TestCase
 {
+    public function setUp(): void
+    {
+        parent::setUp();
+        
+        set_error_handler(
+            static function ($errno, $errstr) {
+                throw new \ErrorException($errstr, $errno);
```

Comment:
The constructor of `\ErrorException` is not equivalent to that of `\RuntimeException`:
https://www.php.net/manual/en/class.errorexception.php#errorexception.examples
```suggestion
            static function ($errno, $errstr, $errfile, $errline) {
                throw new \ErrorException($errstr, 0, $errno, $errfile, $errline);
```